README.md

```diff
@@ -1,2 +1,21 @@
 # open-source-funders
-Name and descrition TBA
+
+> Name and description TBA
+
+## Background
+
+Funding in open source is a continual problem. For a project to survive beyond the momentary interest of its maintainer, it needs funds. Today, there are more ways of funding open source projects and contributors than ever: GitHub Sponsors, Open Collective, npm.fund, Patreon, and so on. Thousands of projects have made it clear they need funds; hundreds of sponsors have helped out. But it's not always clear what companies do sponsor Open Source, and which companies don't. It's also not obvious which companies sponsor who on what platform. And while tens of thousands of companies have GitHub organization profiles, we suspect that only a few hundred companies actively sponsor the code that they use.
```

Comment:
is it `npm.fund`?

Reply:
`what companies do sponsor Open Source`, is `which` better?

Reply:
I’d drop the `and which companies don't`

Reply:
I’d drop `And while tens of thousands of companies have GitHub organization profiles, we suspect that only a few hundred companies actively sponsor the code that they use.` as well.